### PR TITLE
StagingAPI: complex delete requests are no longer used

### PR DESCRIPTION
**Pull request by DimStar77**
* Simplify the 'accept' code

openSUSE:Factory/snapshot is newly setup as a Download on Demand repository
and we can thus safe ourselves the hassle of having to take extra care of
handling delete requests. They no longer bleed directly into the /snapshot
area and can now be accepted like any other request

* Drop vdelreq plugin

Since we no longer do a virtual accept of delete requests, we also don't
have to monitor their state anymore. Drop the plugin that served only
this usecase

* Drop the acheck command

As a consequence, the command 'staging acheck' serves no purpose anymore:
/totest was just there to ensure we have something to copy to /snapshot.
This entire repo will vanish, and it is thus no longer needed to verify
if it is in sync before accepting new data.

### CODE REVIEW

File: osclib/accept_command.py [hunk 1]
```diff
@@ -137,19 +89,13 @@ def perform(self, project, force=False):
...
-                self.virtually_accept_delete(req['id'], req['package'])
-            else:
-                print(msg)
-                change_request_state(self.api.apiurl,
-                                     str(req['id']),
-                                     'accepted',
-                                     message='Accept to %s' % self.api.project,
-                                     force=force)
-                self.create_new_links(self.api.project, req['package'], oldspecs)
+            print(msg)
```
Comment (coolo) [inline osclib/accept_command.py]: if there is only one msg left - just print it

File: dist/package/openSUSE-release-tools.spec [hunk 2]
```diff
@@ -305,16 +305,6 @@ Requires:       osclib = %{version}
 %description -n osc-plugin-staging
 OSC plugin for the staging workflow, see `osc staging --help`.
 
-%package -n osc-plugin-vdelreq
```
[the remark this answers is not in the document]
Comment (DimStar77) [reply dist/package/openSUSE-release-tools.spec]: Heard of them :) - but: which package should obsolete it? (opensuse-release will do it already)

[the remark this answers is not in the document]
Comment (DimStar77) [reply dist/package/openSUSE-release-tools.spec]: Obsoleted by main package

File: osclib/accept_command.py (hunk 1 above)
Comment (DimStar77) [reply osclib/accept_command.py]: PR updated

**Comment by DimStar77** [comment]
> LGTM. Maybe we should also remove delreq-review setting from conf.py?

Added a commit cleaning up the config

**Pull request merged**